placeSpecificContent/hq/courtyard/entityResources/podium/courtyardPodium_server.js

```diff
@@ -0,0 +1,173 @@
+                openedLocalPosition = { x: closedLocalPosition.x, y: -0.9608, z: closedLocalPosition.z };
+            } else if (name === "Bingo Stage Entry Gate") {
+                closedLocalPosition = { x: 1.0367, y: 1.3896, z: 0.0816 };
+                openedLocalPosition = { x: closedLocalPosition.x, y: -0.8328, z: closedLocalPosition.z };
+            } else if (name === "Prize Door 1 Gate") {
+                closedLocalPosition = { x: -2.8130, y: -0.8786, z: -1.2755 };
+                openedLocalPosition = { x: closedLocalPosition.x, y: -3.6787, z: closedLocalPosition.z };
+            } else if (name === "Prize Door 2 Gate") {
+                closedLocalPosition = { x: 0.0067, y: -0.8784, z: -1.2755 };
+                openedLocalPosition = { x: closedLocalPosition.x, y: -3.6787, z: closedLocalPosition.z };
+            } else if (name === "Prize Door 3 Gate") {
+                closedLocalPosition = { x: 2.8165, y: -0.8782, z: -1.2755 };
+                openedLocalPosition = { x: closedLocalPosition.x, y: -3.6787, z: closedLocalPosition.z };
+            }
+            
+            Entities.editEntity(_this.entityID, { localPosition: closedLocalPosition});
+        },
+
+        // Gate moves DOWN to open
+        openGate: function() {
+            // Clear check interval if one exists.
+            if (gatePositionCheckInterval) {
+                Script.clearInterval(gatePositionCheckInterval);
+                gatePositionCheckInterval = false;
+            }
+
+            // Check if already open.
+            var currentPosition = Entities.getEntityProperties(_this.entityID, 'localPosition').localPosition;
+            if (currentPosition.y >= openedLocalPosition.y - EPSILON_M &&
+                currentPosition.y <= openedLocalPosition.y + EPSILON_M) {
+                return;
+            }
+
+            playSound(OPEN_SOUND, Entities.getEntityProperties(_this.entityID, 'localPosition').localPosition, 1);
+
+            // Start opening the gate.
+            Entities.editEntity(_this.entityID,
+                {
+                    velocity: {x: 0, y: -MOVEMENT_VELOCITY_M_PER_SEC, z: 0}
+                }
+            );
+
+            // Start the check interval that stops the gate when it's open.
+            gatePositionCheckInterval = Script.setInterval(function() {
+                var localPosition = Entities.getEntityProperties(_this.entityID, 'localPosition').localPosition;
+                if (localPosition.y <= openedLocalPosition.y) {
+                    Entities.editEntity(_this.entityID,
+                        {
+                            localPosition: openedLocalPosition,
+                            velocity: {x: 0, y: 0, z: 0}
+                        }
+                    );
+                    Script.clearInterval(gatePositionCheckInterval);
+                    gatePositionCheckInterval = false;
+                }
+            }, POSITION_CHECK_INTERVAL_MS);
+        },
+
```

Comment:
This file might need to be looked at, I think there are a lot of Bingo left overs.